alot: 0.9 → 0.9.1

milibopp opened the pull request:
Dropped the patch, as it is included in the point release.

Switched the hash representation to regular base32.

###### Motivation for this change

New release came out. There are a few bug fixes in it including but not limited to the patch that was already in nixpkgs.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

milibopp committed:
alot: 0.9 → 0.9.1

Dropped the patch, as it is included in the point release.

Switched the hash representation to regular base32.

geistesk commented:
Thanks for this change. Would you like to become the maintainer of this package? I'm no longer a user of `alot` and don't have the resources to _maintain_ this package. Feel free to alter `maintainers` and change our names or at least drop mine.

jonringer commented on pkgs/development/python-modules/alot/default.nix:
```diff
@@ -16,18 +16,9 @@ buildPythonPackage rec {
     repo = "alot";
     rev = version;
-    sha256 = "sha256-WUwOJcq8JE7YO8sFeZwYikCRhpufO0pL6MKu54ZYsHI=";
+    hash = "sha256:0s94m17yph1gq9f2svipb3bbwbw1s4j3zf2xkg5h91006v8286r6";
```
most standard way in nixpkgs
```suggestion
    sha256 = "0s94m17yph1gq9f2svipb3bbwbw1s4j3zf2xkg5h91006v8286r6";
```

aepsil0n replied:
Oh, really? Curiously [someone on IRC](https://logs.nix.samueldr.com/nixos/2020-05-14#3458998;) implied that would not be the best way to go, but no worries I'll change it.

emilazy replied:
The `hash` form is for SRI-style hashes (see the [Nix 2.2 release notes](https://nixos.org/nix/manual/#ssec-relnotes-2.2)), is the default output of new enough Nix, and will be preferred in the future, so it's fine to use.

jonringer replied:
oh interesting, sorry, didn't know

aepsil0n commented:
> Thanks for this change. Would you like to become the maintainer of this package? I'm no longer a user of `alot` and don't have the resources to _maintain_ this package. Feel free to alter `maintainers` and change our names or at least drop mine.

Yes, I use it as my daily driver at least for private mails, so I definitely want alot to keep running smoothly on NixOS. I will update the maintainers field accordingly.

milibopp committed:
alot: replace geistesk as maintainer

As requested by geistesk here:
https://github.com/NixOS/nixpkgs/pull/87803#issuecomment-628711513

aepsil0n commented:
I changed the first commit to use the proper hash and added a second to take over as maintainer from @geistesk. So the PR should be ready to merge now.
